### ISSUE TITLE
Publish Gradle plugin to the Plugin Portal

### ISSUE THREAD

**Issue by rachelcarmena** [arrow-kt/arrow-meta#41]
The configuration to publish Arrow Meta artifacts (snapshots and releases) in Bintray is being added in #40.

However, Gradle Plugin is not included in #40 because it has its own way to be published to the Plugin Portal and also its own version roadmap:

https://plugins.gradle.org/docs/publish-plugin

The desired name for the Gradle plugin would be:

```
io.arrow-kt.arrow
```

**Comment by rachelcarmena** [comment]
The plugin has been published:

* https://plugins.gradle.org/plugin/io.arrow-kt.arrow

and the artifact of the plugin is available successfully:

![Screenshot from 2019-10-31 18-13-30](https://user-images.githubusercontent.com/22792183/67969787-3b85ae80-fc0a-11e9-9a66-aa28e03ca2f5.png)

However, we're going to change the groupId `arrow.meta.plugin.gradle` by `io.arrow-kt`

**Comment by rachelcarmena** [comment]
When fixing the groupId, it's considered as a **new** plugin so we must wait up to 24 hours for the new publication.

**Comment by rachelcarmena** [comment]
The plugin is published again: https://plugins.gradle.org/plugin/io.arrow-kt.arrow

However, I'm solving a problem with a hardcoded version in `ArrowKotlinGradleSubplugin` which raises this error when using the plugin:

```
Could not find io.arrow-kt:gradle-plugin:0.0.1.
```

**Comment by rachelcarmena** [comment]
I'm going to close this issue because Gradle plugin is published, new versions will be published without required approval from Gradle team and the problems found when using the published plugin have been solved in #69.
